src/python/tests/appengine/handlers/cron/service_accounts_test.py

```diff
@@ -22,15 +22,15 @@ class ServiceAccountIdTest(unittest.TestCase):
   """Tests _service_account_id."""
 
   def test_min_length(self):
-    self.assertEqual(service_accounts._service_account_id('a'), 'bot-a0')
+    self.assertEqual(service_accounts._service_account_id('a'), 'bot-a0')  # pylint: disable=protected-access
 
   def test_regular_length(self):
-    self.assertEqual(service_accounts._service_account_id('abc'), 'bot-abc')
+    self.assertEqual(service_accounts._service_account_id('abc'), 'bot-abc')  # pylint: disable=protected-access
 
   def test_max_length(self):
     self.assertEqual(
-        service_accounts._service_account_id('a' * 26), 'bot-%s' % ('a' * 26))
+        service_accounts._service_account_id('a' * 26), 'bot-%s' % ('a' * 26))  # pylint: disable=protected-access
 
   def test_more_than_max_length(self):
     with self.assertRaises(AssertionError):
-      service_accounts._service_account_id('a' * 27)
+      service_accounts._service_account_id('a' * 27)  # pylint: disable=protected-access
```

Comment:
In this case it might be better to do this at the file-level.

Reply:
I think we generally do it multiple times per file. But I can change it here if you'd still like me to.